sdk/android/skin/src/main/java/com/ooyala/android/skin/OoyalaSkinLayout.java

```diff
@@ -94,45 +101,71 @@ public int getViewHeight() {
   }
 
   public boolean isFullscreen() {
-    return isFullscreen;
+    return fullscreen;
```

Comment:
Yes what I mean is that in other parts of the code we use `this.fullscreen` when referring to the field of the class and just `fullscreen` when it's a parameter. My suggestion is just to use `this.fullscreen` here in order to be consistent.